mapbox/services/tokens.py

```diff
@@ -23,8 +23,73 @@ def create(self, username, scopes=None, note=None, expires=0):
         payload = {'scopes': scopes, 'note': note}
 
         if expires > 0:
-            payload['expires'] = arrow.now().replace(seconds=+expires).isoformat()
+            payload['expires'] = (datetime.now() + timedelta(seconds=expires)).isoformat()
 
         res = self.session.post(uri, json=payload)
         self.handle_http_error(res)
         return res
+
+    def list_tokens(self, username, limit=None):
+        uri = URITemplate(self.baseuri + '/{username}').expand(username=username)
+
+        params = {}
+        if limit:
+            params['limit'] = int(limit)
+
+        res = self.session.get(uri, params=params)
+        self.handle_http_error(res)
+        return res
+
+    def create_temp_token(self, username, scopes=None, expires=0):
+        if not scopes:
+            raise ValueError("One or more token scopes are required")
+
+        uri = URITemplate(self.baseuri + '/{username}').expand(username=username)
+
+        payload = {'scopes': scopes}
+
+        if 0 < expires <= 3600:
+            payload['expires'] = (datetime.now() + timedelta(seconds=expires)).isoformat()
+        else:
+            raise ValueError("Expiry should be within 1 hour from now")
+
+        res = self.session.post(uri, json=payload)
+        self.handle_http_error(res)
+        return res
+
+    def update_auth(self, username, authorization_id, scopes=None, note=None):
```

Comment:
For this method, and all of the methods which take `username` as a parameter, we could make it an optional keyword argument. Since the username is embedded in the access token itself, we can assume that as the default.

See https://github.com/mapbox/mapbox-sdk-py/blob/master/mapbox/services/base.py#L59

In this case we'll need to shuffle the input arguments. Here's a rough sktech of my idea:
```
def update_auth(self, authorization_id, username=None, scopes=None, note=None):
    if username is None:
        username = self.username
```

Let's use this pattern to provide a default username to this and all the other Token methods.